recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "2.4.0.dev" %}
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/2.4-rc2.tar.gz
+  sha256: 254eb82717ffec8b79618c281fe5c87734b2f8d804ef369d414d91b1b71cd163
+
+build:
+  number: 0
+  skip: true  # [win32]
+  msvc_compiler: 14.0
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - toolchain
+    - python
+    - numpy
+    - swig
+    - hdf5 1.8.18|1.8.18.*
+    - zlib 1.2.*
+    - libmatio >=1.5.12
+    - clangdev 4.0*  # [unix]
+    - libcxx 4.0.*  # [unix]
+    - openmp 4.0.*  # [unix]
+    - libgfortran  # [unix]
+    - openblas 0.2.20|0.2.20.*
```

Comment:
This doesn't need an explicit pinning any more. It is fixed externally and this constraint is added to the packages automatically.

ref: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1f9b4021bfaa830eef6786ac373b9b10478aa2da/recipe/conda_build_config.yaml#L165-L166
ref: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1f9b4021bfaa830eef6786ac373b9b10478aa2da/recipe/conda_build_config.yaml#L346-L347